### PR TITLE
Copy anchor to clipboard

### PR DESCRIPTION
**Pull request by dbreseman**

### CODE REVIEW

File: assets/js/anchor.js [hunk 1]
```diff
@@ -1,46 +1,64 @@
...
+                placement: 'right',
+                trigger: 'hover focus'
+            });
+
+            // Add on-click behavior
+            a.addEventListener('click', (event) => {
+                event.preventDefault(); // Prevent scrolling
+                const href = a.href;
+                navigator.clipboard.writeText(href); // Copy to clipboard
+                window.history.pushState({}, document.title, href); // Update URL
```
Comment (MarkvanMents) [inline assets/js/anchor.js:45]: Should we change the URL if we aren't scrolling there?
Perhaps talk to UX about what their preferred behavior is?

**Review by MarkvanMents — COMMENTED** [review]
Hi Dana (@dbreseman)
This looks really great.
Two things which are not important, but might be nice to address.

1. See note about browser URL, below.

2. We still have this issue that there seems to be a space below headings which are nearly the whole width of the window

image
Where the purpose only becomes clear when you mouse over and the link icon appears
image
I don't know if there is an aesthetic solution to this?

Thanks for picking this up so quickly.

File: assets/js/anchor.js (hunk 1 above)
Comment (dbreseman) [reply assets/js/anchor.js:45]: Hi, thanks for the feedback.
- Extra spacing: Ah no, I had this resolved but must have reintroduced it when I changed the tooltip type. I'll take a look and see if I can eliminate this.
- URL: I think it does make sense to change the URL—if the user is copying something to clipboard, I think it's nice if they can see what it is. This is also what Confluence does. I'm not deeply attached to it though, and can certainly check in with UX on this—do you know who the right contact person or Slack channel would be?

Comment (dbreseman) [reply assets/js/anchor.js:45]: @MarkvanMents Fixed now! If this looks good to you, I'll go ahead and merge, and then add any adjustments from the UX team once they've had a chance to review as well.

Comment (MarkvanMents) [reply assets/js/anchor.js:45]: Thanks Dana:
* Extra spacing: solved (although I had to go to a private window to stop my cached javascript triggering!). Thanks.
* URL: let's publish it as it is, but talk to UX (Ashley?) about it. Normally what is in the URL reflects what is being displayed on the page. If we change the URL without moving the anchor to the top then that breaks the (my😀) paradigm. But perhaps it isn't a problem for UX and most people?

**Review by MarkvanMents — APPROVED** [review]
@dbreseman 
Thanks for this. We can follow up the UX issue with the URL later.

File: assets/js/anchor.js (hunk 1 above)
Comment (dbreseman) [reply assets/js/anchor.js:45]: Sounds good! I reached out to Ashley, so see what she thinks.

**Pull request merged**